compute/functions/reference-content/code-examples.mdx

```diff
@@ -896,5 +898,66 @@ function handle($event, $context) {
 		"statusCode" => 200,
 	];
 }
+```
+
+## Getting data
+**Using environment variables**
+
+Python runtime includes the `os`, which enables you to access the environment variables you define in your namespaces or functions.
+
+The following snippet returns the environment variable `ENV` you specified in the function or namespace configuration:
+```php
+<?php
+
+function handle($event, $context) {
+	$env = getenv('ENV');
+	return [
+		"body" => $env,
+		"statusCode" => 200,
+	];
+}
+```
+
+### Using event objects
+
+With event objects you can pass information through your HTTP request. They are composed of the following objects:
+
+* `pathParameters`: map(string)string - Parameters defined in the path of the HTTP Request
+* `queryStringParameters`: map(string)string - Query Strings parameters of the HTTP Request
+* `body`: string|byte() - Body of the HTTP Request, you will have to parse it in your handler to use it properly.
+* `headers`: map(string)string - HTTP Request Headers
+* `method`: string - HTTP method used
+* `isBase64Encoded`: boolean - Whether the request body is base64 encoded.
+
+
+<?php
+
+function handle($event, $context) {
+    var_dump($event);
+    $queryparameter = (str)$event["queryStringParameters"]["parameter"];
+ 
+    $querybody = $event["body"];
+
```

Comment:
remove this line

Reply:
the empty line 👌

Reply:
@lucasscw is it ok to resolve this conversation and merge the PR?